### PR TITLE
Fixing gold standard script to work with circle ci

### PR DESCRIPTION
**Pull request by chummels**
This should fix issue #132 so the gold standard script now work with circle_ci instead of travis.

### CODE REVIEW

**Comment by chummels** [comment]
Actually, no, this doesn't address everything.  This merely fixes the breakage of the script in accessing the travis yml file.  But all of the other text in the script that helped you actually build the gold standard was removed, presumably during the switch to circle ci.  I'm not sure why.  Was there something wrong with the build order that was given to the user described in the docs: https://trident.readthedocs.io/en/latest/testing.html#generating-gold-standard-answer-test-results-for-comparison ?  It's useful to be explicit with the user in regenerating thegold standards, and I personally find it helpful when I haven't built the gold standards in a long time to remember the steps.

File: tests/gold_standard_versions.py [hunk 1]
```diff
@@ -13,22 +13,25 @@
...
+        if match_YT:
+            yt_gold = match_YT.group(1)
+        if match_trident:
+            trident_gold = match_trident.group(1)
     f.close()
     print()
     print('Latest Gold Standard Commit Tags\n')
-    print('yt = %s' % yt_gold, end='')
-    print('Trident = %s' % trident_gold)
+    print('yt = %s' % yt_gold)
```
Comment (brittonsmith) [inline tests/gold_standard_versions.py:33]: If you initialize `yt_gold` and `trident_gold` to None, you can check that they are not None at the end and then run this script with a test. That would prevent future breakages.

**Comment by brittonsmith** [comment]
I'm not sure what other text you're referring to. Between this script and the docs you link to, I don't see anything else that one needs to know to run testing. The build script is written to most efficiently use the testing workflow set up by the service, nothing more. It's not meant to be used as documentation. I missed this because it's not tested functionality. I left a comment on how it could be.

**Comment by chummels** [comment]
@brittonsmith You're right.  I had mistakenly thought that the text that was in the docs about how to generate the gold standards was also meant to be printed to STDOUT as part of this script.  But I looked back at past versions and that was never the case.  Sorry about that.

**Comment by chummels** [comment]
I have now updated the script to include a `test_gold.py` file that will ensure that this functionality is tested in the future.  Although I don't foresee us changing the testing infrastructure again, but I guess more tests are always better.

File: tests/gold_standard_versions.py (hunk 1 above)
Comment (chummels) [reply tests/gold_standard_versions.py:35]: Done and done.

**Pull request merged**